### PR TITLE
📷 Add camera rotate methods

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

Add camera rotate methods
- `roll`
- `vertical_rotate`
- `horizontal_rotate`

 
https://github.com/pyvista/pyvista-support/issues/412

### Details

This is a wrapper of `Roll`, `Elevation` and `Azumith` of [`vtkCamera` class](https://vtk.org/doc/nightly/html/classvtkCamera.html).

camera

### CODE REVIEW

File: pyvista/plotting/camera.py [hunk 1]
```diff
@@ -340,3 +340,81 @@ def view_frustum(self, aspect=1.0):
...
+        focal point as the center of rotation.
+
+        Parameters
+        ----------
+        angle : float
+           Rotate angle.
+
+        Returns
+        -------
+        None
```
Comment (akaszynski) [inline pyvista/plotting/camera.py:387]:
```suggestion
```

File: pyvista/plotting/camera.py [hunk 2]
```diff
@@ -340,3 +340,81 @@ def view_frustum(self, aspect=1.0):
...
+        and is not necessarily perpendicular to the direction of projection.
+
+        Parameters
+        ----------
+        angle : float
+           Rotate angle.
+
+        Returns
+        -------
+        None
```
Comment (akaszynski) [inline pyvista/plotting/camera.py:411]:
```suggestion
```

File: pyvista/plotting/camera.py [hunk 3]
```diff
@@ -340,3 +340,81 @@ def view_frustum(self, aspect=1.0):
...
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> pl = pyvista.Plotter()
+        >>> pl.camera.vertical_rotate(45.0)
+        """
+        self.Elevation(angle)
+
+    def horizontal_rotate(self, angle):
```
Comment (akaszynski) [inline pyvista/plotting/camera.py:397]: Can we make this a setter/getter like `roll`?

File: pyvista/plotting/camera.py [hunk 4]
```diff
@@ -340,3 +340,81 @@ def view_frustum(self, aspect=1.0):
...
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> pl = pyvista.Plotter()
+        >>> pl.camera.roll = 45.0
+        """
+        self.SetRoll(angle)
+
+    def vertical_rotate(self, angle):
```
Comment (akaszynski) [inline pyvista/plotting/camera.py:373]: Can we make this a setter/getter like `roll`?

File: pyvista/plotting/camera.py (hunk 3 above)
Comment (tkoyama010) [reply pyvista/plotting/camera.py:393]: Yes. In that case, we should calculate `horizontal_rotate` in pyvista. Please wait while I make it.

File: pyvista/plotting/camera.py (hunk 4 above)
Comment (tkoyama010) [reply pyvista/plotting/camera.py:373]: Yes. In that case, we should calculate `vertical_rotate` in pyvista. Please wait while I make it.

**Comment by akaszynski** [comment]
I've made some changes, most notably, renaming `vertical_rotate` to `elevation` and `horizontal_rotate` to `azimuth`.  I figured it was more inline with VTK's API and other common camera methods.

Also, I've added some additional docs for clarity.  For example: 

https://user-images.githubusercontent.com/11981631/116963481-31f95e80-ac66-11eb-9599-5176fa0d1624.mp4


```py
import pyvista
from pyvista import demos
pl = demos.orientation_plotter()
pl.camera_position = 'yz'
pl.camera.elevation = 45
pl.show()
```

image

**Review by akaszynski — APPROVED** [review]
Approved!  Please let me know if you agree with my changes.  As always, thanks for all your hard work.

**Review by tkoyama010 — COMMENTED** [review]
@akaszynski Thanks. I am a fan of this camera explanation. As always, thanks for all your great review. It is a good chance for me to improve my skill of developing.

**Pull request merged**